<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="CAPHYD-LAP-081" ignored="0" name="First_XML_Program" tests="3" failures="2" timestamp="04 Sep 2019 16:58:29 GMT" time="0.118" errors="0">
  <testcase name="validation" time="0.018" classname="Pratise.FirstTestNG">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Failing this becasue of overcome expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Failing this becasue of overcome expected [false] but found [true]
at Pratise.FirstTestNG.validation(FirstTestNG.java:30)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- validation -->
  <testcase name="Regist" time="0.004" classname="Pratise.Dependecy">
    <failure type="java.lang.AssertionError" message="Failed">
      <![CDATA[java.lang.AssertionError: Failed
at Pratise.Dependecy.Regist(Dependecy.java:10)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- Regist -->
  <testcase name="Login" time="0.012" classname="Pratise.Dependecy"/>
</testsuite> <!-- First_XML_Program -->
